src/components/ResearchTile.vue

```diff
@@ -33,13 +35,19 @@ import Resource from './Resource.vue';
     federation(): Federation {
       if (this.level == 5 && this.field === ResearchField.Terraforming) {
         return this.$store.state.gaiaViewer.data.terraformingFederation;
+      }    
+    },
+    lostPlanet(): PlanetEnum {
+      if (this.level == 5 && this.field === ResearchField.Navigation) {
```

Comment:
This should check that no one actually reached the end of the track. The lost planet should disappear if someone else is on the research tile

Reply:
done